src/jose/types.ts

```diff
@@ -0,0 +1,45 @@
+export interface Jwk {
+  alg?: string;
+  kid?: string;
+}
+
+/**
+ * A SECP256K1 public key in JWK format.
+ * Values taken from:
+ * https://www.iana.org/assignments/jose/jose.xhtml#web-key-elliptic-curve
+ * https://datatracker.ietf.org/doc/html/draft-ietf-cose-webauthn-algorithms-06#section-3.1
+ */
+export type JwkEd25519Public = Jwk & {
+  alg: 'EdDSA';
```

Comment:
Sanity: did three of us agree to not require `alg`? Okay as long as it doesn't blow up when it is not supplied by DID resolution.

Reply:
we did. can make it optional here as well. Didn't think it was going to make much of a difference if we made it required here since it's confined to this codebase and doesn't imply any runtime checks. Not a fan of explosions so i went ahead and made it optional: https://github.com/TBD54566975/dwn-sdk-js/commit/f9ab6086604c1499eb8cc699d9d492ba0a705113